docs/src/pages/guides/aliases.md

```diff
@@ -3,36 +3,47 @@ layout: ~/layouts/MainLayout.astro
 title: Aliases
 ---
 
-Aliasing is an integral part when it comes to clean and readable code, that's why Astro comes with support for custom aliases out of the box.
+An alias is a handy shortcut to your paths. Projects can get quite messy when it comes to file structure. Astro gives you the ability to define _keywords_ which you can use no matter how deep your file is located. 
```

Comment:
nit: `to your import paths`.

also: `Projects can get quite messy when it comes to file structure.` is still too opinionated :) How about this?

```suggestion
An **alias** is a handy shortcut for your JavaScript import paths. This can be a great option if you dislike long relative import paths with many repeating `../` segments. Define an **alias** in your project to import directly from some directory no matter how deeply nested a file is located. 
```